libtransmission/peer-mgr.cc

```diff
@@ -317,7 +317,18 @@ class tr_swarm
     tr_swarm(tr_peerMgr* manager_in, tr_torrent* tor_in) noexcept
         : manager{ manager_in }
         , tor{ tor_in }
+        , tags_{ {
```

Comment:
May be a good idea to use `tor_in` instead of `tor` just in case you ever move the declaration of `tags_` above `tor`.

Reply:
done in aefacb6799a8a6fb463a258f29baba5032c24875